tsconfig.base.json

```diff
@@ -14,6 +14,8 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
question: how does this affect our production build? Any gotchas?

Reply:
esModuleInterop will change how typescript treats CommonJS/AMD/UMD: https://www.typescriptlang.org/tsconfig#esModuleInterop

I don't have any gothas about resolveJsonModule besides actually being able to import `.json`

Reply:
Let's try it out but stay aware if any issues surface.